packages/core/src/plugins/popup-controller.class.ts

```diff
@@ -1,11 +1,22 @@
 import { AuguryWindow } from '../augury-window.interface';
 
+interface WindowSizeAndPosition {
+  position: { x: number; y: number };
+  size: { width: number; height: number };
+}
+
 export class PopupController {
   public readonly window: AuguryWindow;
+  protected storageKey: string;
```

Comment:
Does this need to be protected for any particular reason that we can think of right now? If not, I'd make it private for now.

Reply:
I think you are right, I'll make it private.